Earthfile

```diff
@@ -466,7 +466,7 @@ earthly-docker:
 # Otherwise, it will attempt to login to the docker hub mirror using the provided username and password
 earthly-integration-test-base:
     FROM +earthly-docker
-    RUN apk update && apk add pcre-tools curl python3 bash perl findutils expect yq
+    RUN apk update && apk add pcre-tools curl python3 bash perl findutils expect yq && apk add --upgrade sed
```

Comment:
I think it's worthwhile adding a comment why the `--upgrade` flag is needed here -- is it because the included version of sed is busybox, which doesn't work? (I had to run this target to come up with this guess)

Reply:
Yes, that is exactly the reason